### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.7 es 4.25 (30 ottobre)

### DISCUSSION THREAD

**Discussion by erinrrr** [sapienzastudentsnetwork/calcolo-delle-probabilita#312] [Esercizi M-Z] [answered]
![image](https://github.com/user-attachments/assets/40a43c56-138e-4618-9d06-276eeeb251b9)

**Comment by erinrrr** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/81319abe-f8fe-4461-a991-2e420d64ee72)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/55d319cf-a735-4848-a279-e4a512fe8086)
